src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,62 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final int ZEROELEMENT = 0;
+    private static final int FIRSTELEMENT = 1;
+    private static final String COMMA = ",";
```

Comment:
COMMA doesn't make it more informative, you should see extracting variable to a constant as an opportunity to convey its purpose to the reader of your class - this serves as a delimiter